### PR TITLE
Log get_validators_no_cache

### PR DESCRIPTION
**Pull request by hweawer**

### CODE REVIEW

File: src/providers/http_provider.py [hunk 1]
```diff
@@ -93,7 +93,14 @@ def _get(
 
         for host in self.hosts:
             try:
-                return self._get_without_fallbacks(host, endpoint, path_params, query_params, stream)
+                data = self._get_without_fallbacks(host, endpoint, path_params, query_params, stream)
+                logger.debug({
+                    'msg': 'Get request',
+                    'url': endpoint,
+                    'data': f'{data}',
```
Comment (vgorkavenko) [inline src/providers/http_provider.py:100]: Data from `GET` endpoints can be quite large. Validators data and keys info can be as large as `300mb` (or even larger). Are we sure we want to print it? @F4ever.

Comment (hweawer) [reply src/providers/http_provider.py]: Logging only a single request, as discussed

File: src/providers/consensus/client.py [hunk 2]
```diff
@@ -160,6 +160,7 @@ def get_validators(self, blockstamp: BlockStamp) -> list[Validator]:
     def get_validators_no_cache(self, blockstamp: BlockStamp, pub_keys: str | tuple | None = None) -> list[dict]:
         """Spec: https://ethereum.github.io/beacon-APIs/#/Beacon/getStateValidators"""
         try:
+            logger.info({'msg': 'Getting validators...'})
```
Comment (F4ever) [inline src/providers/consensus/client.py:163]: blockstamp.slot_number
blockstamp.state_root
Add:
url without domen

Comment (F4ever) [reply src/providers/consensus/client.py:163]: Move outside of try block

Comment (F4ever) [reply src/providers/consensus/client.py:163]: Add final log with response details + with validators length

**Review by vgorkavenko — APPROVED** [review]

**Review by F4ever — APPROVED** [review]

**Pull request merged**